package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Vulcan",
-  "version": "1.2.0",
+  "name": "LessWrong",
+  "version": "0.0.0",
```

Comment:
Since we're not planning on publishing this on npm (we have private=true set), I left it with the placeholder version of 0.0.0